src/ApiInterface/include/ApiInterface/Orbit.h

```diff
@@ -353,13 +356,13 @@ struct orbit_api_v1 {
   void (*track_double)(const char* name, double value, orbit_api_color color);
 };
 
-extern struct orbit_api_v1 g_orbit_api_v1;
-extern ORBIT_EXPORT void* orbit_api_get_function_table_address_v1();
+extern struct orbit_api_v2 g_orbit_api_v2;
```

Comment:
I wonder if the variable should have a more generic name, `g_orbit_api`, version bumps would then modify less code, wdyt? I can't remember if there was a special reason not to do this on the first version bump.

Reply:
I would say that one advantage of having the version in the variable name is that, well, the version is explicit and it can help us not make mistakes in handling the versions. But you're right that it's not necessary, the version (and soon the platform) is determined by `orbit_api_get_function_table_address_v#`.
I'll follow up with a separate pull request as I would want to keep the commits separate anyway.